packages/stark-ui/src/modules/date-range-picker/components/date-range-picker.component.html

```diff
@@ -1,26 +1,24 @@
-<div class="stark-date-range-picker">
-    <stark-date-picker 
-        #startPicker
-        [pickerId]="rangePickerId+'-start'" 
-        [pickerName]="rangePickerName+'-start'" 
-        [date]="startDate" 
-        [dateFilter]="dateFilter"
-        [isDisabled]="isDisabled" 
-        [label]="startDateLabel" 
-        [minDate]="startMinDate" 
-        [maxDate]="startMaxDate"
-        (dateChanged)="onDateStartChanged($event)">
-    </stark-date-picker>
-    <stark-date-picker
-        #endPicker 
-        [pickerId]="rangePickerId+'-end'" 
-        [pickerName]="rangePickerName+'-end'" 
-        [date]="endDate" 
-        [dateFilter]="dateFilter"
-        [isDisabled]="isDisabled" 
-        [label]="endDateLabel" 
-        [minDate]="endMinDate" 
-        [maxDate]="endMaxDate" 
-        (dateChanged)="onDateEndChanged($event)">
-    </stark-date-picker>	
-</div>
+<stark-date-picker
```

Comment:
Ok it is better to remove the parent div. But it break te layout in the showcase, on the example viewer for this component. The checkbox is now on the same line and too close to the end date picker.
Can you fix that?

Reply:
Ok. I'll fix that. I didn't see it.